src/main/webAuthnService.ts

```diff
@@ -84,6 +84,7 @@ export type RegistrationPublicKeyCredentialSerialized = {
   response: {
     attestationObject: string
     clientDataJSON: string
+    transports: string[]
```

Comment:
```suggestion
    transports: ReturnType<typeof AuthenticatorAttestationResponse.prototype['getTransports']>
```
Simple suggestion qui peut éviter des surprises en cas d'évolution de la norme ou des types fournis pas TypeScript..

Reply:
Si je comprends bien: `transports` a pour type le type de retour de la méthode `getTransports` de `AuthenticatorAttestationResponse` ? Il n'y a pas de notation autre que `AuthenticatorAttestationResponse.prototype['getTransports']>`?

Reply:
C'est ça. Cette notation permet de reprendre le type tel qu'il est déclaré sur le retour de `getTransport`, soit : 
```("ble" | "hybrid" | "internal" | "nfc" | "usb")[]```
Ce qui est plus précis que simplement `string[]`